packages/app/src/cli/models/app/app.ts

```diff
@@ -350,7 +350,7 @@ export class App<
           type: module.externalType,
           handle: module.handle,
           uid: module.uid,
-          assets: module.uid,
+          assets: module.handle,
```

Comment:
This change doesn't affect the current behavior in Partners?

Reply:
no, this is the manifest, is not used anywhere